_data/tools.yml

```diff
@@ -1218,6 +1218,14 @@
   description: OpenAPI 3.0 (and Swagger v2) implementation for Go (parsing, converting, validation, and more)
   v3: true
 
+- name: openapi.tanna.dev/go/validator
+  category:
+    - data-validators
+  github: https://gitlab.com/jamietanna/httptest-openapi/
```

Comment:
Even though this isn't on GitHub, this should be fine, right?

Reply:
We should change this to source code. @mbifulco can you consider this for the rewrite?

Reply:
I'll change it to `repo:` !